python/reg_interface/scripts/repeated_reg_read.py

```diff
@@ -0,0 +1,139 @@
+
+    f.write( "|        | CRC_ERROR_CNT | PACKET_ERROR_CNT | BITSTUFFING_ERROR_CNT | TIMEOUT_ERROR_CNT | AXI_STROBE_ERROR_CNT | TRANSACTION_CNT |\n")
+    print( "|        | CRC_ERROR_CNT | PACKET_ERROR_CNT | BITSTUFFING_ERROR_CNT | TIMEOUT_ERROR_CNT | AXI_STROBE_ERROR_CNT | TRANSACTION_CNT |")
+    
+    f.write("|:------ | :------------ | :--------------- | :-------------------- | :---------------- | :------------------- | :-------------- |\n")
+    print("|:------ | :------------ | :--------------- | :-------------------- | :---------------- | :------------------- | :-------------- |")
+
+    f.write("| before | {0}  | {1} | {2} | {3} | {4} | {5} |\n".format(
+            crc_error_cnt_before,
+            packet_error_cnt_before,
+            bitstuffing_error_cnt_before,
+            timeout_error_cnt_before,
+            axi_strobe_error_cnt_before,
+            transaction_cnt_before
+         ))
+
+    print("| before | {0}  | {1} | {2} | {3} | {4} | {5} |".format(
+            crc_error_cnt_before,
+            packet_error_cnt_before,
+            bitstuffing_error_cnt_before,
+            timeout_error_cnt_before,
+            axi_strobe_error_cnt_before,
+            transaction_cnt_before
+         ))
+
+    node = getNode(args.reg_name)
+
+    register_values = {}
+    
+    for i in range(0,int(args.nreads)):
+        value = readAddress(node.real_address)
+
+        if value in register_values.keys():
+            register_values[value] += 1
+        else:
+            register_values[value] = 1
+        
+        time.sleep(float(args.sleeptime)/float(1000000))
+
+    crc_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.CRC_ERROR_CNT").real_address)
+    packet_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.PACKET_ERROR_CNT").real_address)
+    bitstuffing_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.BITSTUFFING_ERROR_CNT").real_address)
+    timeout_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.TIMEOUT_ERROR_CNT").real_address)
+    axi_strobe_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.AXI_STROBE_ERROR_CNT").real_address)
+    transaction_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.TRANSACTION_CNT").real_address)
+
+    crc_error_cnt_delta = str(int(crc_error_cnt_after,16) - int(crc_error_cnt_before,16))
+    packet_error_cnt_delta = str(int(packet_error_cnt_after,16) - int(packet_error_cnt_before,16))
+    bitstuffing_error_cnt_delta = str(int(bitstuffing_error_cnt_after,16) - int(bitstuffing_error_cnt_before,16))
+    timeout_error_cnt_delta = str(int(timeout_error_cnt_after,16) - int(timeout_error_cnt_before,16))
+    axi_strobe_error_cnt_delta = str(int(axi_strobe_error_cnt_after,16) - int(axi_strobe_error_cnt_before,16))
+    transaction_cnt_delta = str(int(transaction_cnt_after,16) - int(transaction_cnt_before,16))
```

Comment:
Why are you doing `str(int(value,base))`?
In the format string you can control this anyway, are you worried about the return type of `readAddress`?